### PR TITLE
miccb-5571 miccb 10.2 power bi templates

### PR DESCRIPTION
**Pull request by shrubs-ok**
Adding the power bi templates of 10.2 for general availability

Epics :
MiCCB-1836 IVR reports
MiCCB-1837 Multimedia reports
MiCCB-2081 Readonly Config

### CODE REVIEW

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Readonly Configuration
+This repository contains Power BI templates providing a readonly view of configuration
```
Comment (shaapala) [inline MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: grammer police :P

should be 
read-only

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Readonly Configuration
```
Comment (shaapala) [inline MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: Read-Only

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+## Configuration_IVR_TechPreview
+* This template is a read-only view of MiContact Center Business IVR configuration. 
+
+
+The storage mode for this report dashboard is Direct
+### Collect the Database information
+Before you can use the template in your environment you need to collect some information about the MiCC-B Server.
+* The Instance name of the Sql Server
+* A username and password which can be used to connect to SQL Server
+* The authenticattion type for the connection ie. SQL or Windows
```
Comment (shaapala) [inline MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: spelling:
authentication

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+### Determine the Connectivity Mode 
+Power BI supports two types of Connectivity: Import and Direct. The decision of which option is based on your specific company policy.  Things that may affect this decision include
+* Are you using licensed version of Power BI?
+* Are you connecting to live MiCC-B system?
+* Are you connecting to a data warehouse?
+* How often do you need the data to be refreshed
+
+The two supported connectivity modes are Import and Direct. 
+* Import pulls data into Power BI and requires refreshing. 
+* Direct is a direct connnection to the live database.
```
Comment (shaapala) [inline MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: spelling:
connection

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+The two supported connectivity modes are Import and Direct. 
+* Import pulls data into Power BI and requires refreshing. 
+* Direct is a direct connnection to the live database. 
+
+There are pros and cons to both of these choices. It is recommended you review both options extensively to ensure you are not affecting the performance of MiCC-B.
+
+
+### Use This Template in Your Environment 
+To use this template in Your Enviroment follow these steps:
```
Comment (shaapala) [inline MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: spelling:
Environment

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+* Set SQL Server Database
+* Select Connect
+
+#### SQL Authentication
+
+If the authentication type is SQL
+* Go To **File**: **Options and Settings**
+* Select the correct SQL Server 
+* Click **Edit Permissions**
+* Select **Database** from the left hand panel
```
Comment (shaapala) [inline MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: left-hand

**Review by shaapala — CHANGES_REQUESTED** [review]
Some grammar and spelling issues

File: MiCCDataAnalytics/PowerBI/Mulitmedia/Readme.md
Comment (shaapala) [inline MiCCDataAnalytics/PowerBI/Mulitmedia/Readme.md:1]: same spelling issues in this one as above to fix

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md (hunk 1 above)
Comment (shrubs-ok) [reply MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: done

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md (hunk 2 above)
Comment (shrubs-ok) [reply MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: done

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md (hunk 3 above)
Comment (shrubs-ok) [reply MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: done

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md (hunk 4 above)
Comment (shrubs-ok) [reply MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: done

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md (hunk 5 above)
Comment (shrubs-ok) [reply MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: done

File: MiCCDataAnalytics/PowerBI/Configuration/Readme.md (hunk 6 above)
Comment (shrubs-ok) [reply MiCCDataAnalytics/PowerBI/Configuration/Readme.md]: done

**Comment by shrubs-ok** [comment]
Change implemented as requested across all three readme files. Noted suggestion to us word for future proofing

File: MiCCDataAnalytics/PowerBI/Mulitmedia/Readme.md
Comment (shrubs-ok) [reply MiCCDataAnalytics/PowerBI/Mulitmedia/Readme.md:1]: done

**Review by shaapala — APPROVED** [review]
Looks good

**Pull request merged**